mitosheet/mitosheet/experiments/experiment_utils.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+"""
+Contains utilities for interacting with experiments. If you want to change the currently running
+experiment, do the following:
+
+1. Change the get_new_experiment() function to return the new experiment you want to run.
+2. Change the user.json version to increase by one.
+3. Write an upgrader function that changes the experiment in the user json to the new experiment.
+
```

Comment:
TODO: leave a note about tracking the ids somewhere, and what the variants mean. So we always know!